packages/evm-adapter/src/abi/helio.ts

```diff
@@ -4,136 +4,14 @@ export const helio = {
-        },
-        {
-          indexed: true,
-          internalType: 'address',
-          name: 'recipient',
-          type: 'address',
-        },
-        {
-          indexed: false,
-          internalType: 'uint256',
-          name: 'transferAmount',
-          type: 'uint256',
-        },
-        {
-          indexed: false,
-          internalType: 'uint256',
-          name: 'feeAmount',
-          type: 'uint256',
-        },
-        {
-          indexed: false,
-          internalType: 'string',
-          name: 'transactionDbId',
-          type: 'string',
-        },
-      ],
-      name: 'EthPayment',
-      type: 'event',
-    },
-    {
-      anonymous: false,
-      inputs: [
-        {
-          indexed: true,
-          internalType: 'address',
-          name: 'sender',
-          type: 'address',
-        },
-        {
-          indexed: true,
-          internalType: 'address',
-          name: 'recipient',
-          type: 'address',
-        },
-        {
-          indexed: true,
-          internalType: 'address',
-          name: 'tokenAddress',
-          type: 'address',
-        },
-        {
-          indexed: false,
-          internalType: 'uint256',
-          name: 'transferAmount',
-          type: 'uint256',
-        },
-        {
-          indexed: false,
-          internalType: 'uint256',
-          name: 'feeAmount',
-          type: 'uint256',
-        },
-        {
-          indexed: false,
-          internalType: 'string',
-          name: 'transactionDbId',
-          type: 'string',
-        },
-      ],
-      name: 'Payment',
-      type: 'event',
-    },
-    {
-      anonymous: false,
-      inputs: [
-        {
-          indexed: true,
-          internalType: 'address',
-          name: 'sender',
-          type: 'address',
-        },
-        {
-          indexed: true,
-          internalType: 'address',
-          name: 'recipient',
-          type: 'address',
-        },
-        {
-          indexed: false,
-          internalType: 'uint256',
-          name: 'transferAmount',
-          type: 'uint256',
-        },
-        {
-          indexed: false,
-          internalType: 'uint256',
-          name: 'feeAmount',
-          type: 'uint256',
-        },
-        {
-          components: [
-            {
-              internalType: 'address',
-              name: 'recipient',
-              type: 'address',
-            },
-            {
-              internalType: 'uint256',
-              name: 'amount',
-              type: 'uint256',
-            },
-          ],
-          indexed: false,
-          internalType: 'struct IHelio.RecepientAndAmount[]',
-          name: 'splitData',
-          type: 'tuple[]',
-        },
         {
           indexed: false,
-          internalType: 'string',
-          name: 'transactionDbId',
-          type: 'string',
+          internalType: 'uint8',
+          name: 'version',
+          type: 'uint8',
```

Comment:
alternatively you could have left the old ABI unchanged and versioned this one, that way we can reference the old contract